content/administration/maintain/azure_oauth.rst

```diff
@@ -0,0 +1,170 @@
+
+:guilabel:`Search` and :guilabel:`Add permissions` for the following :guilabel:`Deregulated
+permissions`:
+
+- :guilabel:`SMTP.Send`
+- :guilabel:`IMAP.AccessAsUser.All`
+
+The :guilabel:`User.Read` permission will be added by default.
+
+.. image:: azure_oauth/permissions.png
+   :align: center
+   :alt: API permissions added for the app.
+
+Assign Users and Groups
+=======================
+
+Navigate back to the :guilabel:`Overview` of the :guilabel:`Application` in the top of the menu on
+the left-hand side. It's time to add users onto this application. Under the :guilabel:`Essentials`
+overview table click on the link :guilabel:`Managed Application in Local Directory`, or the last
+option on the bottom right-hand side of the table.
+
+.. image:: azure_oauth/managed-application.png
+   :align: center
+   :alt: Link to the created application and to add users/groups.
+
+In the :guilabel:`Left Menu` select :guilabel:`Users and Groups`. Next select the :guilabel:`(+)
+Add User/Group`. Depending on the account, either a :guilabel:`Group` and :guilabel:`User` can be
+added, or :guilabel:`Users` alone. Personal accounts will only allow for users to be added.
+
+Add the users or group of users that will be sending from the :guilabel:`Microsoft account` in
+:guilabel:`Odoo` by clicking on :guilabel:`None Selected` under users or groups. :guilabel:`Add`
+the users/groups, click :guilabel:`Select` and then :guilabel:`Assign` them to the application.
+
+This completes the setup on the :guilabel:`Microsoft Azure Portal` side.
+
+Setup in Odoo
+=============
+
+Enter Microsoft Outlook Credentials
+-----------------------------------
+
+First, open :guilabel:`Odoo` and navigate to the :guilabel:`Apps module`. Then, remove the Apps
+filter from the :guilabel:`search bar` and type in `Outlook`. Install the module called
+:guilabel:`Microsoft Outlook`.
+
+Next, navigate to :guilabel:`Settings ‣ General Settings`, and under the :guilabel:`Discuss`
+section, ensure that the checkbox for :guilabel:`Custom Email Servers` is checked. This populates a
+new option for :guilabel:`Outlook Credentials`.
+
+:guilabel:`Save` the progress.
+
+Then, copy and paste the :guilabel:`Client ID (Application ID)` and :guilabel:`Client Secret
+(Client Secret Value)` into the respective fields and :guilabel:`Save` the settings.
+
+.. image:: azure_oauth/outlookcreds.png
+   :align: center
+   :alt: Outlook Credentials in Odoo General Settings.
+
+Configure outgoing email server
+-------------------------------
+
+After that, on the :guilabel:`General Settings` page, under :guilabel:`Custom Email Servers`, click
+:guilabel:`Outgoing Email Servers` to configure the external :guilabel:`Microsoft account`.
+
+Then, create a new email server and select the option for :guilabel:`Outlook`. Next, fill in the
+:guilabel:`Name` (it can be anything) and the Microsoft Outlook email :guilabel:`Username`. If
+empty, enter the :guilabel:`From Filter`.
```

Comment:
```suggestion
Then, create a new email server and check the box for :guilabel:`Outlook`. Next, fill in the
:guilabel:`Description` (it can be anything) and the Microsoft Outlook email :guilabel:`Username`.
If empty, enter the :guilabel:`From Filter`.
```

Nitpick: this PR is targeting the `14.0` branch, and in Runbot 14.0, this field is called "Description" instead of "Name." I would make note of this so you can open a new PR to correct it for 15.0 user docs, after this PR is merged and forward-ported.

Also, question @tiku-odoo:  by `` If empty, enter the :guilabel:`From Filter`. `` what do you mean? If the "Username" is empty? It's unclear.

Reply:
Note: [Task #3048351](https://www.odoo.com/web#id=3048351&cids=3&menu_id=4720&action=333&active_id=6050&model=project.task&view_type=form)